src/components/common/SummaryOfficeAddresses.vue

```diff
@@ -120,8 +120,8 @@ export default class SummaryOfficeAddresses extends Mixins(CommonMixin, EntityFi
   @Prop({ default: null })
   private recordsAddress: BaseAddressObjIF
 
-  // Entity Enum
-  private EntityTypes: {} = EntityTypes
+  // Enum definition for use in template.
+  private EntityTypes = EntityTypes
```

Comment:
FYI, this is so we don't lose type information. (Previously type was {}.)

Reply:
Definitely cleaner! If only there was a way of binding the enum directly to the class without having to define a property of the same name on the class.

Reply:
This is only needed for the template to see the types. The script doesn't need it. Just a Vue thing I guess.